### PR TITLE
FINERACT-1870: Run reports fix for 1.8.x

### PR DESCRIPTION
**Pull request by vidakovic**
Run report fix for 1.8.x

### CODE REVIEW

File: fineract-provider/src/main/java/org/apache/fineract/infrastructure/security/utils/SQLInjectionValidator.java [hunk 1]
```diff
@@ -37,23 +38,32 @@ private SQLInjectionValidator() {
...
+    // https://larrysteinle.com/2011/02/20/use-regular-expressions-to-detect-sql-code-injection more examples
+    private static final List<String> INJECTION_PATTERNS = List.of("(?i).*[or|and]\s*[\"']?-1[\"']?\\s*(-*).*",
+            "(?i).*\\s+[\"']?(\\d+)[\"']?\\s*=\\s*[\"']?(\\1)[\"']?\\s*(-*).*");
+
     public static void validateSQLInput(final String sqlSearch) {
         if (StringUtils.isBlank(sqlSearch)) {
             return;
         }
+
+        // TODO: this should be replaced by INJECTION_PATTERNS
```
Comment (adamsaghy) [inline fineract-provider/src/main/java/org/apache/fineract/infrastructure/security/utils/SQLInjectionValidator.java:51]: Is the TODO for future?
Should we replace it now?

Comment (vidakovic) [reply fineract-provider/src/main/java/org/apache/fineract/infrastructure/security/utils/SQLInjectionValidator.java:51]: Well... maybe I should remove it here. This a maintenance release, so changes should be as minimal as possible. But for the upcoming upstream develop PR (not there yet) we could discuss this a bit more. There are definitely more cases to cover.

Here I am just trying to do the necessary to fix the issue. So, if you are ok with that then let's move the discussion to upstream develop and leave things as they are here. If you want I can remove the TODOs.

**Review by adamsaghy — APPROVED** [review]
LGTM

**Comment by adamsaghy** [comment]
would you mind to write a test to cover the fixed behaviour?

**Pull request merged**